model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -589,6 +589,45 @@ class LicenseInfoResolverTest : WordSpec({
             result should containCopyrightStatementsForLicenseExactly("LicenseRef-a")
             result should containCopyrightStatementsForLicenseExactly("LicenseRef-b")
         }
+
+        "resolve copyrights from authors in concluded license" {
```

Comment:
Also here, you're not testing something that's specific to disclosure documents, so your commit message should not mention that. Any our test is in the model, not the reporter.

Reply:
PS: I liked the previous approach with disabled test being added beforehand better, but this can stay as-is now from my perspective.

Reply:
Adapted the commit message.